src/util/codecs.js

```diff
@@ -74,23 +74,28 @@ export const codecsForPlaylist = function(master, media) {
 
   // HLS with multiple-audio tracks must always get an audio codec.
   // Put another way, there is no way to have a video-only multiple-audio HLS!
-  if (isMaat(master, media) && !codecInfo.audioProfile) {
+  if (isMaat(master, media) && !codecInfo.audioCodec) {
     if (!isMuxed(master, media)) {
       // It is possible for codecs to be specified on the audio media group playlist but
       // not on the rendition playlist. This is mostly the case for DASH, where audio and
       // video are always separate (and separately specified).
-      codecInfo.audioProfile = audioProfileFromDefault(master, mediaAttributes.AUDIO);
+      const defaultCodecs = codecsFromDefault(master, mediaAttributes.AUDIO);
```

Comment:
this function uses `parseCodecs` and the function name itself was changed to be more generic.